templates/includes/pagination.html

```diff
@@ -0,0 +1,15 @@
+<ul class="pagination">
+  {% if post_list.has_previous %}
+    <li class="page-item">
+      <a href="?page={{ post_list.previous_page_number }}" class="page-link">prev</a>
+    </li>
+  {% endif %}
+  <li class="page-item active">
+    <span class="page-link">{{ post_list.number }} of {{ paginator.num_pages }}</span>
+  </li>
+  {% if post_list.has_next %}
+    <li class="page-item">
+      <a href="?page={{ post_list.next_page_number }}" class="page-link">next</a>
```

Comment:
The 'next' link should be disabled or hidden when there is no next page. Consider adding a condition to check if 'post_list.has_next' is true before rendering this link.